rllib/core/learner/torch/torch_learner.py

```diff
@@ -436,41 +428,14 @@ def build(self) -> None:
         """Builds the TorchLearner.
 
         This method is specific to TorchLearner. Before running super() it will
-        initialze the device properly based on the `_use_gpu` and `_distributed`
-        flags, so that `_make_module()` can place the created module on the correct
-        device. After running super() it will wrap the module in a TorchDDPRLModule
-        if `_distributed` is True.
+        initialize the device properly based on `self.config`, so that `_make_module()`
+        can place the created module on the correct device. After running super() it
+        wraps the module in a TorchDDPRLModule if `config.num_learners > 0`.
```

Comment:
`config.num_learners > 0`  -> `self.distributed`?

Reply:
Yeah, just trying to reduce the number of weird, hidden flags, which have no real use. Using the config imo is more explicit, transparent, and explicatory.